axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveLoadDefaultConfigServiceImpl.java

```diff
@@ -67,24 +67,26 @@ public Account getAccountingAccountFromAccountConfig(Move move) {
   public TaxLine getTaxLine(Move move, MoveLine moveLine, Account accountingAccount) {
     TaxLine taxLine = null;
     Partner partner = move.getPartner();
-    if (ObjectUtils.isEmpty(partner.getFiscalPosition())) {
-      if (accountingAccount != null && accountingAccount.getDefaultTax() != null) {
-        taxLine = accountingAccount.getDefaultTax().getActiveTaxLine();
-        if (taxLine == null || !taxLine.getStartDate().isBefore(moveLine.getDate())) {
-          taxLine =
-              findValidTaxLineForMoveLine(
-                  accountingAccount.getDefaultTax().getTaxLineList(), moveLine);
-        }
-      }
-    } else {
-      for (TaxEquiv taxEquiv : partner.getFiscalPosition().getTaxEquivList()) {
-        if (accountingAccount != null
-            && taxEquiv.getFromTax().equals(accountingAccount.getDefaultTax())) {
-          taxLine = taxEquiv.getToTax().getActiveTaxLine();
+    if (!ObjectUtils.isEmpty(partner)) {
+      if (ObjectUtils.isEmpty(partner.getFiscalPosition())) {
+        if (accountingAccount != null && accountingAccount.getDefaultTax() != null) {
+          taxLine = accountingAccount.getDefaultTax().getActiveTaxLine();
```

Comment:
There is already an existing method to get the taxLine

https://github.com/axelor/axelor-open-suite/blob/master/axelor-base/src/main/java/com/axelor/apps/base/service/tax/TaxService.java#L53